contracts/axelar-interchain-token-service/src/error.rs

```diff
@@ -0,0 +1,12 @@
+use soroban_sdk::contracterror;
+
+#[contracterror]
+#[derive(Copy, Clone, Debug, Eq, PartialEq, PartialOrd, Ord)]
```

Comment:
i forget whether soroban errors require all of these. `Ord` is an interesting one to require for error types

Reply:
Hmm let me check on that...

Reply:
According to Soroban's example contract explanation: 
"Contract errors are Rust u32 enums where every variant of the enum is assigned an integer. The #[contracterror] attribute is used to set the error up so it can be used in the return value of contract functions.

The enum has some constraints:

    It must have the #[repr(u32)] attribute.
    It must have the #[derive(Copy)] attribute.
    Every variant must have an explicit integer value assigned.
"

Not sure why so many traits are included in their example. Let me see if there's better docs somewhere else.